sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -582,9 +582,10 @@ def _make_metric_list_to_collect(self, custom_metrics):
                     metrics_to_collect.append(self.typed_metric(cfg_inst=cfg, table=table, column=column))
 
         # Load DB File Space Usage metrics
-        for name, table, column in DATABASE_FILE_SPACE_USAGE_METRICS:
-            cfg = {'name': name, 'table': table, 'column': column, 'instance_name': 'tempdb', 'tags': tags}
-            metrics_to_collect.append(self.typed_metric(cfg_inst=cfg, table=table, column=column))
+        if is_affirmative(self.instance.get('include_db_file_space_usage_metrics', True)):
```

Comment:
Can we rename the configuration option to `include_tempdb_file_space_usage_metrics` ? Just so users know it's specific to tempdb?